src/app/docs/page.tsx

```
@@ -1,254 +1,279 @@
-                </code>
-              </div>
-            </div>
-
-            <div className="border border-gray-200 rounded-lg p-6">
-              <div className="flex items-center space-x-3 mb-4">
-                <div className="bg-blue-100 p-2 rounded-lg">
-                  <span className="text-blue-600 font-semibold">2</span>
+                <div className="bg-gray-900 text-gray-100 p-4 rounded-lg mb-4">
+                  <code
+                    className="text-sm whitespace-pre-wrap"
+                    dangerouslySetInnerHTML={{ __html: step.command }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)